components/micro-gateway-core/src/main/ballerina/src/gateway/constants/api_security_constants.bal

```diff
@@ -29,7 +29,7 @@ public const string API_AUTH_INVALID_CREDENTIALS_DESCRIPTION =
 public const int API_AUTH_MISSING_CREDENTIALS = 900902;
 public const string API_AUTH_MISSING_CREDENTIALS_MESSAGE = "Missing Credentials";
 public const string API_AUTH_MISSING_CREDENTIALS_DESCRIPTION =
-"Make sure your API invocation call has a header: \"Authorization: Bearer ACCESS_TOKEN\"";
+"Make sure your API request provides required credentials";
```

Comment:
Try to use the APIM message for the consistancy

Reply:
APIM message is "Make sure your API invocation call has a header". I believe this message is kind of invalid now.